openwisp_firmware_upgrader/base/models.py

```diff
@@ -162,7 +164,7 @@ def batch_upgrade(self, firmwareless, upgrade_options=None):
         batch.full_clean()
         batch.save()
         transaction.on_commit(
-            lambda: batch_upgrade_operation.delay(batch.pk, firmwareless)
+            partial(batch_upgrade_operation.delay, batch.pk, firmwareless)
```

Comment:
I just would like to understand if there was a need to change this or if you just wanted to rewrite it?
If there was a need, what was it?